bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
-            "CO2_Level": {},
-            "PM10_Level": {},
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        QUDT.applicableUnit: [
+            UNIT["CentiM"],
+            UNIT["DeciM"],
+            UNIT["MilliM"],
+            UNIT["MicroM"],
+            UNIT["KiloM"],
+            UNIT["M"],
+            UNIT["IN"],
+            UNIT["FT"],
+            UNIT["YD"],
+        ],
+        QUDT.hasDimensionVector: QUDTDV["A0E0L1I0M0H0T0D0"],
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Level"),
+        SKOS.broader: QUDTQK.Length,
+        "subconcepts": {
+            "Precipitation": {
+                QUDT.applicableUnit: [
+                    UNIT["CentiM"],
+                    UNIT["DeciM"],
+                    UNIT["MilliM"],
+                    UNIT["MicroM"],
+                    UNIT["KiloM"],
+                    UNIT["M"],
+                    UNIT["IN"],
+                    UNIT["FT"],
+                    UNIT["YD"],
+                ],
+                SKOS.definition: Literal(
+                    "Amount of atmospheric water vapor fallen including rain, sleet, snow, and hail (https://project-haystack.dev/doc/lib-phScience/precipitation)"
+                ),
+                QUDT.hasDimensionVector: QUDTDV["A0E0L1I0M0H0T0D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Precipitation"),
+                SKOS.broader: QUDTQK.Length,
+            },
         },
     },
-    "Luminance": {"subclasses": {"Luminous_Flux": {}, "Luminous_Intensity": {}}},
-    "Occupancy": {"subclasses": {"Occupancy_Count": {}, "Occupancy_Percentage": {}}},
-    "Position": {},
-    "Power_Factor": {},
-    "Precipitation": {},
+    "Luminance": {
+        OWL.sameAs: QUDTQK["Luminance"],
+        "subconcepts": {
+            "Luminous_Flux": {OWL.sameAs: QUDTQK["LuminousFlux"]},
+            "Luminous_Intensity": {OWL.sameAs: QUDTQK["LuminousIntensity"]},
+        },
+    },
+    "Occupancy": {
+        "subconcepts": {
+            "Occupancy_Count": {
+                QUDT.applicableUnit: [UNIT["People"]],
+                SKOS.definition: Literal("Number of people in an area"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Occupancy_Count"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "Occupancy_Percentage": {
+                QUDT.applicableUnit: [UNIT["PERCENT"]],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal(
+                    "Percent of total occupancy of space that is occupied"
+                ),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Occupancy_Percentage"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+        }
+    },
+    "Position": {
```

Comment:
Do we have other use cases of this than Damper_Position? The term "Position" is very generic and can mislead users to a different meaning (a person's position) than what's intended here. I'd rather just use Angle for Damper Position and remove this.

Reply:
There's a definition attached that clarifies that this is not for a person's position: do you think the definition needs work? This is *definitely* not a "rate". Position may be used for other moving parts such as valves

Reply:
The word Position is not only used for moving parts represented by a percentage but location too.

Reply:
Ok, but that can be a distinct term, e.g. "Location" or "coordinates". We are providing a definition here of what we mean (and what practitioners commonly mean) as Position